AnnotationParser.js

```diff
@@ -223,17 +234,16 @@ class AnnotationParser {
 			// not yet sure what to do with effects 
 			// const effects = appearanceElement.getAttribute("effects");
 
-			const styles = {};
 			// 0.00 to 1.00
-			if (bgOpacity) styles.bgOpacity = parseFloat(bgOpacity, 10);
+			styles.bgOpacity = bgOpacity ? parseFloat(bgOpacity, 10) : styles.bgOpacity;
```

Comment:
The ternary should probably be removed. The else case isn't really doing anything since `styles.bgOpacity` is already set to the default.
I just used:
```javascript
if (bgOpacity) ...
```

Reply:
:+1:, fixed